### PR TITLE
Flux 2020 Annual Review

### PR DESCRIPTION
**Pull request by squaremo**
Please find enclosed the Flux project annual review for year ending August 2020.

### CODE REVIEW

File: reviews/2020-flux-annual.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+## Does Flux meet the criteria for incubation?
+
+We think so.
+
+ * Adoption continues to grow, and we see more and more talks, how-to
+   guides and integrations around.
+ * Cloud vendors and industry partners integrate with Flux and are
+   watching GOTK development closely.
+ * As Flux is in a very interesting phase of its development right
```
Comment (alena1108) [inline reviews/2020-flux-annual.md:141]: Could you elaborate on "some while" - is it months, years? Or is it something that can't be estimated at this moment.

File: reviews/2020-flux-annual.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+ * GitHub stars have gone from about 2500 to >5300 in the last year;
+ * we believe (based on optional telemetry in fluxd) that the install
+   base is on the order of tens of thousands of instances.
+
+### Performance against goals
+
+This is the Flux project's first review, so there are no goals already
+on record. However, there are some measures of progress:
+
+ * We have been talking to users and customers a lot. The Flux
```
Comment (alena1108) [inline reviews/2020-flux-annual.md:91]: Developer advocacy for GitOps in the community is the great effort. Thank you for doing that!

File: reviews/2020-flux-annual.md (hunk 1 above)
Comment (squaremo) [reply reviews/2020-flux-annual.md:141]: Low numbers of months. There's a timeline [here](https://toolkit.fluxcd.io/roadmap/); we're operating a migration window during which both Flux v1 and v2 are maintained, but expect contributions to Flux v1 to dwindle quickly.

File: reviews/2020-flux-annual.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+   community was instrumental in running the [GitOps
+   Days](https://www.gitopsdays.com/) event, for example.
+ * We started experimentation in the GitOps Toolkit, designed the APIs
```
Comment (alena1108) [inline reviews/2020-flux-annual.md:94]: Can we also include the link to the Flux2 roadmap? Assuming it's this link https://toolkit.fluxcd.io/roadmap/#the-road-to-flux-v2; if not - please include the most recent one

Comment (dholbach) [reply reviews/2020-flux-annual.md:94]: That's the one!

Comment (squaremo) [reply reviews/2020-flux-annual.md:94]: I've included a link to the roadmap directly below this.

**Comment by alena1108** [comment]
LGTM for Sandbox annual review

**Comment by dholbach** [comment]
JFYI https://github.com/cncf/toc/pull/567 is up for review now too…

**Comment by amye** [comment]
This was accepted in the December 8 meeting.

**Pull request merged**